ballerina-tests/http-interceptor-tests/tests/test_common.bal

```diff
@@ -373,6 +373,16 @@ service class RequestInterceptorUserAgentField {
     }
 }
 
+service class RequestInterceptorJwtInformation {
+    *http:RequestInterceptor;
+
+    resource function 'default [string... path](http:RequestContext ctx, http:Request req) returns http:NextService|error? {
+        reqCtxJwtValues[0] = check ctx.getWithType(http:JWT_HEADER);
```

Comment:
Shouldn't we introduce new function(s) on Request Context object to retrieve these information(this will forbid users to override these internal values)  rather than using the members? We can have a function like `getJWTInfo()` and return `[jwt:Header, jwt:Payload]|error`.
@chamil321 @shafreenAnfar WDYT?

Reply:
If we use a separate function, then we can set the jwt string as an attribute of the request context object and do the decoding in the ballerina side :
```bal
public isolated class RequestContext {
    private final string? JWTString = (); // Need to set this value from the native side

    ......

    function getJWTInfo() returns [jwt:Header,  jwt:Payload]|error? {
        if self.JWTString is string {
            // Have to return a HttpError by adding the jwt:Error as cause
            return jwt:decode(<string> self.JWTString);
        }
        return ();
    }
}
```

Reply:
Updated as discussed